src/ActionsMenuToggle.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react';
+import { injectIntl } from 'react-intl';
+import type {IntlShape} from 'react-intl';
+
+import { ReactComponent as EllipsisIcon } from './svg/Ellipsis.svg';
+
+import './ActionsMenuToggle.scss';
+
+type ActionsMenuToggleProps = {
+    intl: IntlShape,
+    editingDisabled: boolean,
+    handleShowHideMenu: () => void
+};
+
+class ActionsMenuToggle extends React.Component<ActionsMenuToggleProps, {}> {
+    /* istanbul ignore next */
+    handleKeyDown = (e: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (e.key === ' ' || e.key === 'Enter') {
+            e.preventDefault();
+            this.props.handleShowHideMenu();
+        }
+    }
+
+    onClick = (e: SyntheticEvent<HTMLElement>) => {
+        e.preventDefault();
+        this.props.handleShowHideMenu();
+    }
+
+    render() {
+        return (
+            <div
```

Comment:
We need a way to communicate that this menu is open or collapsed to screen readers. For that, this toggle menu needs aria-expanded and aria-controls attributes. We have used the attributes in the program blocks and the action panel.

Reply:
I made a go at this based on that example.